docker-compose.override.apache-druid.yml

```diff
@@ -0,0 +1,101 @@
+volumes:
+
+  common-middlemanager-var:
+    name: 'common-middlemanager-var'
+
+  common-historical-var:
+    name: 'common-historical-var'
+
+  common-broker-var:
+    name: 'common-broker-var'
+
+  common-coordinator-var:
+    name: 'common-coordinator-var'
+
+  common-router-var:
+    name: 'common-router-var'
+
+  common-druid-shared:
+    name: 'common-druid-shared'
+
+
+services:
+
+  coordinator:
+    image: apache/druid:${APACHE_DRUID_IMAGE_TAG}
+    container_name: common-coordinator
+    depends_on: 
+      - zookeeper
+      - postgres
+    command:
+      - coordinator
+    env_file:
+      - .envs/apache-druid.env
+    networks:
+      common-net:
+        aliases:
+          - common-coordinator
+
+  broker:
```

Comment:
Add `apache_druid_` prefix

Reply:
Added `apache_druid_` prefix for the relevant service name.